specs/altair/p2p-interface.md

```diff
@@ -161,7 +163,9 @@ The following validations MUST pass before forwarding the `sync_committee_messag
 - _[IGNORE]_ The message's slot is for the current slot (with a `MAXIMUM_GOSSIP_CLOCK_DISPARITY` allowance), i.e. `sync_committee_message.slot == current_slot`.
 - _[REJECT]_ The `subnet_id` is valid for the given validator, i.e. `subnet_id in compute_subnets_for_sync_committee(state, sync_committee_message.validator_index)`.
   Note this validation implies the validator is part of the broader current sync committee along with the correct subcommittee.
-- _[IGNORE]_ There has been no other valid sync committee message for the declared `slot` for the validator referenced by `sync_committee_message.validator_index`
+- _[IGNORE]_ The block being signed (`sync_committee_message.beacon_block_root`) has been seen (via both gossip and non-gossip sources) (a client MAY queue sync committee messages for processing once block is retrieved).
```

Comment:
This is probably pedantic, but `via both gossip and non-gossip` i think more accurately would be reflected as `via gossip or non-gossip sources`
I can't imagine a case where we'd ever want fetching every block that we receive over gossip also over RPC which this currently kind of implies...
This is also repeated above, I think we just change it...

Reply:
This is not new wording from this PR, but instead consistent with existing wording from everywhere else. The semantics here are meant to be identical to the existing one, and wording changes should probably be synchronized across all mentions.

<img width="330" alt="image" src="https://github.com/user-attachments/assets/e290777a-cb95-4151-b2d9-aa1919f719d4" />

Reply:
Ok, i'll raise a separate PR for that, I still think it's misleading.

Reply:
I've fixed the other instances in #4049

Reply:
Thanks, have synchronized the wording to match the one from `dev`.